app/routes/application.js

```diff
@@ -18,40 +18,15 @@ export default Route.extend(ApplicationRouteMixin, {
     this.get('authManager').initialize();
   },
   model() {
-    return RSVP.hash({
-      notifications: [
-        {
-          title       : 'New Session Proposal for event1 by  user1',
-          description : 'Title of proposal',
-          createdAt   : new Date(),
-          isRead      : false
-        },
-        {
-          title       : 'New Session Proposal for event2 by  user2',
-          description : 'Title of proposal',
-          createdAt   : new Date(),
-          isRead      : true
-        },
-        {
-          title       : 'New Session Proposal for event3 by  user3',
-          description : 'Title of proposal',
-          createdAt   : new Date(),
-          isRead      : false
-        },
-        {
-          title       : 'New Session Proposal for event4 by  user4',
-          description : 'Title of proposal',
-          createdAt   : new Date(),
-          isRead      : true
-        },
-        {
-          title       : 'New Session Proposal for event5 by  user5',
-          description : 'Title of proposal',
-          createdAt   : new Date(),
-          isRead      : false
-        }
-      ]
-    });
+    if (this.get('session.isAuthenticated')) {
+      return new RSVP.Promise((resolve, reject) => {
+        this.store.findRecord('user', this.get('authManager.currentUser.id'), { reload: true })
+          .then(user => {
+            user.get('notifications').then(resolve).catch(reject);
```

Comment:
Yes @sumedh123 ... that was due the confusion that I had created in the first place 😓 (Really sorry about that again ...). Then I resolved myself in the latest comment here right -> https://github.com/fossasia/open-event-frontend/pull/570#issuecomment-320183819

> **Quote from the comment linked above:**
>
> *"Ensure the navbar icon always triggers a dropdown and show only unread notifications in the dropdown and have a link in the drop down to the notifications page which would show all notifications"*

To be remove any sort of confusion/doubt/mis-understanding

- Dropdown will show only unread notification
- When no unread notifications are available, show a message in the dropdown saying nothing is available and link to `notifications.all` route 
- When unread notifications are available, show the notifications and also have a link to the `notifications.index` route

(And ... Thank you for being very patient with me on this sumedh 😅 )